python/tests/test_stats.py

```diff
@@ -1584,3 +1584,885 @@ def verify(self, ts, reference_sets, focal=None):
                 mean = np.sum(T * L, axis=1) / np.sum(L)
                 assert np.allclose(mean, A[j])
         return A
+
+
+class TestUnweightedCoalescenceTimeTable:
+    """
+                         Block Weight C.Wght Quant
+                         -------------------------
+    29.00┊    9        ┊    0     1        4  1.00
+         ┊ ┏━━┻━━┓     ┊
+    8.00 ┊ ┃     8     ┊    0     1        3  0.75
+         ┊ ┃   ┏━┻━━┓  ┊
+    5.00 ┊ ┃   7    ┃  ┊    0     1        2  0.50
+         ┊ ┃ ┏━╋━┓  ┃  ┊
+    1.00 ┊ ┃ ┃ ┃ ┃  6  ┊    0     1        1  0.25
+         ┊ ┃ ┃ ┃ ┃ ┏┻┓ ┊
+    0.00 ┊ 0 1 2 4 3 5 ┊    0     0        0  0.00 < to catch OOR
+         0            100
+    Uniform weights on nodes
+    """
+
+    @tests.cached_example
+    def coalescence_time_distribution(self):
+        ts = msprime.sim_ancestry(
```

Comment:
Unfortunately we can't rely on the exact outputs of msprime @nspope. They will differ abitrarily depending on platform and underlying GSL version. 

The simplest thing to do here is probably just to hand code up this example using the tables API. You could also output the text representation of the nodes and edges and store them in the file here, if you prefer (see test_topology.py for lots of examples of this)

Reply:
Working on that now, thanks! It does seem like the issue is the RNG across platforms.